packages/core/src/hooks/useSendTransaction.ts

```diff
@@ -2,18 +2,38 @@ import { TransactionRequest } from '@ethersproject/abstract-provider'
 import { TransactionOptions } from '../../src'
 import { useEthers } from './useEthers'
 import { usePromiseTransaction } from './usePromiseTransaction'
+import { useConfig } from '../providers/config/context'
+import { BigNumber, Signer } from 'ethers'
+
+async function estimateGas(signer: Signer, transactionRequest: TransactionRequest) {
+  let estimatedGas: BigNumber | undefined = undefined
+  try {
+    estimatedGas = await signer.estimateGas(transactionRequest)
+  } catch (err) {
+    console.error('Invalid sender address')
```

Comment:
You have no way of knowing that this is exactly this error? If the estimation throws, let it throw and the user should know.